sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,254 @@
+specification and introduced the identity member function [505aa7d1]
+- Improved `group_local_memory_for_overwrite` (part of 
+[`sycl_ext_oneapi_local_memory`](doc/extensions/supported/sycl_ext_oneapi_local_memory.asciidoc)) 
+by defailt initializing it [37a43f93]
+- Augmented `sycl::has_known_identity` to return true for `std::complex` and 
+`std::plus` operators [2e733125]
+- Improved diagnostic when `sycl::accessor` isn't bound to `sycl::handler` 
+[3c87f55e]
+- Added identities for arithmetic and integral operations on bool in accordance 
+with the SYCL 2020 specification, added a check for the specification defined 
+identities [091481d8]
+- Renamed `raw_send{s}_{load/store}` APIs in ESIMD extension [cb280eab]
+- Allowed const `sycl::ext::oneapi::experimental::device_global` [a43c5bd1]
+- Made `sycl::reducer` uncopyable and immovable in accordance with SYCL 2020 
+[7c7efeee]
+- Updated aspects list to be SYCL 2020 compliant [b9122b30]
+- Updated `sycl::kernel_bundle::has_kernel(kernel_id, device)` to return `true` 
+if the kernel is valid for a given device in accordance with SYCL 2020 
+[a950450d]
+
+### Documentation
+- Added Explicit SIMD design and implementation notes [59dd3879]
+- Removed `sycl_ext_oneapi_nd_range_reductions` docs, implementation had been 
+removed in the middle of 2022 [a46efa0a]
+- Clarified default device_global allocation behavior [15edb1bd]
+- Corrected the specified return type for the UUID query [77d25cd8]
+- Clarified the usage of Intel GPU drivers in WSL under the docker [1f39511c]
+
+## Bug fixes
+
+### SYCL Compiler
+- Fixed `-f[no-]sycl-rdc` option in `clang-cl` [e39d0c18]
+- Fixed emission of FPGA annotations [8199de8c]
+- Cleaned up Windows defaultlib linking behaviors [aaae3348]
+- Fixed an issue where the dimensions of the work-group hint were not correctly 
+reversed [ba51bb68]
+- Fixed a crash during generation of debug information with the option 
+`-ffile-prefix-map` [723b80da]
+
+### SYCL Library
+- Fixed `operator-` for constant accessor iterator [890aa072]
+- Fixed the case when sub-device free memory may exceed root-device free memory 
+[9dc14a29]
+- Fixed `sycl::handler::require()` to accept a non-placeholder argument 
+[abdc0fe3]
+- Fixed `sycl::ext::oneapi::experimental::joint_reduce`, it missed the case when 
+work-group size is bigger than size of input data [23b68dbd]
+- Added missing include of sycl/builtins.hpp to sycl/ext/intel/math.hpp 
+[efe1074e]
+- Fixed host compilation issue for lsc_atomic_update for store operation in 
+ESIMD extension [fdf8ac77]
+- Fixed event profiling for `sycl::info::event_profiling::command_submit` in 
+Level Zero and other backends [71d77972]
+- Fixed `sycl::group::get_local_linear_id()`, now it uses the correct range to 
+linearize the local ID [c3849635]
+- Fixed `sycl::host_accessor` zero dimension constructors [23f107d4]
+- Fixed last group detection's atomic memory_order [2aef0c94]
```

Comment:
```suggestion
- Fixed atomic memory orders in reduction implementation [2aef0c94]
```

Also tagging @aelovikov-intel

Reply:
Either version is fine with me, to be honest :)